run-queue

```diff
@@ -39,10 +40,15 @@ logging.basicConfig(level=logging.INFO)
 
 statistics_queue = os.environ.get("RUN_STATISTICS_QUEUE")
 
-# Returns a command to execute and the delivery tag needed to ack the message
+# as per pika docs
+DeliveryTag = NewType('DeliveryTag', int)
 
 
-def consume_webhook_queue(channel, amqp):
+# Returns a command argv to execute and the delivery tag needed to ack the message
+def consume_webhook_queue(
+    channel: pika.channel.Channel,
+    amqp: str
+) -> tuple[list[str] | None, DeliveryTag | None]:
```

Comment:
Sorry for not catching this before: does this actually have to be `list` or could it be `Sequence`?

If the intent is not for the caller to modify the returned data, `Sequence` is more appropriate.

Reply:
Indeed, here we have a case where order mattes, iteration matters, but the value should be read-only. So as per your list above it should be Sequence.